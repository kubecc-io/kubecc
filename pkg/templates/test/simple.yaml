test:
  string: {{ .Spec.String }}
  int: {{ .Spec.Int }}
  more:
  - data: {{ .Spec.Float }}
