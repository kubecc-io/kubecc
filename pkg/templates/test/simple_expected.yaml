test:
  string: testing
  int: 123
  more:
  - data: 12.3